lib/elixlsx/xml_templates.ex

```diff
@@ -488,6 +489,20 @@ defmodule Elixlsx.XMLTemplates do
     show_grid_lines_xml
   end
 
+  defp make_sheet_show_zeros(sheet) do
+    case sheet.show_zeros do
+      true -> ""
+      false -> ~s[ showZeros="0" ]
+    end
+  end
+
+  defp make_sheet_fit_to_page(sheet) do
+    case sheet.fit_to_page do
+      true -> "1"
+      _ -> "false"
```

Comment:
why is it "1" for the true case, but "false" for the false case? Is it possible to change make both bool or both int? Otherwise, could you add a quick comment on why not?